### PR TITLE
[1LP][RFR] extend travis testing for better verification and py3k strictness

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 236028213fba7534984dfa09bf2008e12d36c710:

`cfme/scripting/setup_env.py`:
- :heavy_exclamation_mark: Line 12:11: **W291** *trailing whitespace*

`cfme/utils/template/ec2.py`:
- :bangbang: Line 3:32: **E401** *multiple imports on one line*

`cfme/utils/template/rhevm.py`:
- :heavy_exclamation_mark: Line 13:9: **W291** *trailing whitespace*

`cfme/utils/trackerbot.py`:
- :bangbang: Line 5:32: **E401** *multiple imports on one line*
- :bangbang: Line 394:101: **E501** *line too long (135 > 100 characters)*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c60736ccd23db2d98ecfdfba98f7f1c0658ed88e:

`cfme/scripting/setup_env.py`:
- :heavy_exclamation_mark: Line 12:11: **W291** *trailing whitespace*

`cfme/utils/template/ec2.py`:
- :bangbang: Line 3:32: **E401** *multiple imports on one line*

`cfme/utils/template/rhevm.py`:
- :heavy_exclamation_mark: Line 13:9: **W291** *trailing whitespace*

`cfme/utils/trackerbot.py`:
- :bangbang: Line 5:32: **E401** *multiple imports on one line*
- :bangbang: Line 394:101: **E501** *line too long (135 > 100 characters)*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/bz.py [hunk 1]
```diff
@@ -27,19 +27,19 @@ def name(self):
...
+        return [release["name"] for release in self._data["releases"]]
 
     @property
     def versions(self):
-        versions = []
-        for version in self._data["versions"]:
-            if version["name"] not in NONE_FIELDS:
-                versions.append(Version(version["name"]))
-        return sorted(versions)
+        return sorted(
```
Comment (mshriver) [inline cfme/utils/bz.py]: :+1:

**Comment by mshriver** [comment]
The auth test failure in PRT is fixed in #7247

**Comment by RonnyPfannschmidt** [comment]
i externalized questionable changes and if this is green it should be good to go

**Pull request merged**